### PR TITLE
feat: ID-1563 Add ERC20 transfer methods to the sample app

### PR DESCRIPTION
**Pull request by imx-mikhala**
# Summary
Adds both the ERC20 `transfer` and the `transferFrom` methods to the sample-app for testing

# Demo

## ERC20 transfer

https://github.com/immutable/ts-immutable-sdk/assets/122326421/89611bbb-1910-432e-a7e3-0b7178189645

## ERC20 transferFrom

https://github.com/immutable/ts-immutable-sdk/assets/122326421/cc2ac0c1-47de-4073-bf83-4dec186d308d

### CODE REVIEW

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/DefaultTransaction.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ function ShowGenericConfirmationScreen({ disabled, handleExampleSubmitted }: Req
   }, [params, handleExampleSubmitted]);
 
   return (
-    <Accordion.Item eventKey="5">
+    <Accordion.Item eventKey="2">
```
Comment (imx-mikhala) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/DefaultTransaction.tsx:83]: just updated these keys so they are in the order they currently appear in accordion

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/DefaultTransaction.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ function ShowGenericConfirmationScreen({ disabled, handleExampleSubmitted }: Req
   }, [params, handleExampleSubmitted]);
 
   return (
-    <Accordion.Item eventKey="5">
+    <Accordion.Item eventKey="1">
```
Comment (imx-mikhala) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/DefaultTransaction.tsx:83]: just updated these keys so they are in the order they currently appear in accordion

File: packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/TransferERC20.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+              disabled={disabled}
+              type="text"
+              onChange={(e) => setToAddress(e.target.value)}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label>
+              Contract Address
+            </Form.Label>
+            <Form.Control
```
Comment (carmen0208) [inline packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/TransferERC20.tsx:145]: we can preset some default contract like Spending cap did: https://github.com/immutable/ts-immutable-sdk/blob/main/packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/SpendingCapApproval.tsx#L33
did

Comment (imx-mikhala) [reply packages/passport/sdk-sample-app/src/components/zkevm/EthSendTransactionExamples/TransferERC20.tsx:160]: done 👍

**Review by carmen0208 — APPROVED** [review]

**Review by yundifu — APPROVED** [review]

**Pull request merged**